its/ruling/src/test/resources/jdk6/squid-S1948.json

```diff
@@ -123,7 +123,6 @@
 81,
 ],
 'jdk6:java/security/SecureRandom.java':[
-78,
```

Comment:
:heavy_check_mark: `private Provider provider = null;` it was a FP `Provider` implements `Serializable` and `null` is also serializable.